packages/cli/src/lib/run/helpers.ts

```diff
@@ -28,11 +28,11 @@ export function revertSortedArgs(processArgs: Array<string>, argv: Array<string>
   return originalInputOrder
 }
 
-export function buildCommand(args: Array<string>) {
+export function buildCommand(args: Array<string>, prependLauncher: boolean = false) {
   if (args.length === 1) {
     // do not add quotes around arguments if there is only one argument
     // `heroku run "rake test"` should work like `heroku run rake test`
-    return args[0]
+    return `${prependLauncher ? 'launcher ' : ''}${args[0]}`
```

Comment:
Rather than doing this twice, maybe declare a `const` and use it here and on line 47. so like:
```javascript
const prependText = prependLauncher ? 'launcher ' : ''
```

Reply:
I'll do that. Great suggestion!